learn/guides/distinctive-language-features/concurrency.md

```diff
@@ -413,7 +413,7 @@ transactional function bar() {
 }
 ```
 
-In the above code example, the named worker **``A``** has the ``transactional`` qualifier. Therefore the strand for execution of the code within **``A``** will have a new transaction that is branched from the calling transactional context within the function **``exec()``**.
+In the above code example, the named worker **``A``** has the ``transactional`` qualifier. Therefore, the strand for execution of the code within **``A``** will have a new transaction that is branched from calling transactional context within the function **``exec()``**.
```

Comment:
Why are we removing "the" here? I believe we need it.

```suggestion
In the above code example, the named worker **``A``** has the ``transactional`` qualifier. Therefore, the strand for execution of the code within **``A``** will have a new transaction that is branched from the calling transactional context within the function **``exec()``**.
```

Reply:
calling is used as a verb; isn't it?

Reply:
No, right? It's "calling transactional context" to mean the transactional context from which it is called?